ci-automation/release.sh

```diff
@@ -94,6 +94,9 @@ function _inside_mantle() {
 
     for platform in aws azure; do
       for arch in amd64 arm64; do
+        if [[ "${arch}" == "arm64" ]] && [[ "${CHANNEL}" == "lts" ]]; then
+          continue
+        fi
```

Comment:
This will also skip aws. Not sure if it's desired. Maybe do something like this:

```suggestion
    for platform in aws azure; do
      arches=(amd64 arm64)
      if [[ "${CHANNEL} = 'lts' ]] && [[ "${platform}" = 'azure' ]]; then
        arches=(amd64)
      fi
      for arch in "${arches[@]}"; do
```